contentcuration/contentcuration/assets/shared/views/PrimaryDialog.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <VDialog
```

Comment:
Note: I'm also working on a shared component, so will want to consolidate eventually: https://github.com/jayoshih/content-curation/blob/vue-edit/contentcuration/contentcuration/static/js/edit_channel/sharedComponents/Dialog.vue

Reply:
I am glad we are having this dialog!